### DISCUSSION TITLE
Dynamically passing passing props to Quasar Vue components in general, and trying to toggle the toolbar on Qeditor in particular

### DISCUSSION THREAD

**Discussion by liconver** [quasarframework/quasar#14075] [General - Components / Directives / etc] [answered]
Hello, basically I was trying different ways to toggle the toolbar on this great Q-editor component so I could just have one component wrapping my editor instead of a bunch of different V-If/Else's in the template.

What I was doing looks something like this codepen https://codepen.io/liconver/pen/abYLgwd though I was using the script setup api with SFC's.

Is it possible to give the ':toolbar=' prop a dynamic value or does it have to be an array in the template? And for the Q-toggle ':false-value' and ':true-value' props could just give them variables as well?

**Comment by MilosPaunovic** [comment, marked as answer]
https://codepen.io/milospaunovic/pen/KKoyPbe?editors=1111

**Reply by liconver** [reply]
Thanks I knew there must have been a simple way to do this.  It's makes sense to keep the toggle model as a boolean and and use a ternary operator for the dynamic people, but it would not have occurred to me to pass a variable that's equal to an array inside of an array.
